app/views/shared/_accordion.html.slim

```diff
@@ -13,8 +13,8 @@
     )
       span.mb0.mr2
         = header_text
-      = image_tag asset_url('plus.svg'), width: 16, class: 'plus-icon display-none'
-      = image_tag asset_url('minus.svg'), width: 16, class: 'minus-icon display-none'
+      = image_tag asset_url('plus.svg'),  alt: 'Plus', width: 16, class: 'plus-icon display-none'
```

Comment:
i18n plz! Include that in the ticket to provide `alt`s

Reply:
I can go ahead an i18N this